_posts/2016-08-01-5-nusmods-features-you-might-not-know-about.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "5 NUSMods Features You Might Not Know About"
+date:   2016-07-31 14:59:26
+author: Tay Yang Shun
+author_thumbnail: 558978353
+---
+
+NUSMods has been an incredibly useful platform for NUS students, providing a user-friendly interface for planning their timetables and searching through the available modules. However, it does more than just that! Here are some of the lesser known features in NUSMods that can improve your productivity and make you even more productive!
```

Comment:
"can improve your productivity and make you even more productive!", repeating, we can say either:
1. "improve your productivity", or
2. "make you even more productive"